gitops/orb.yml

```diff
@@ -0,0 +1,121 @@
+version: 2.1
+description: "Provides jobs/commands for updating Helm chart values and commiting them to Git"
+
+orbs:
+  argocd: ovotech/argocd@1.0.2
+
+executors:
+  python:
+    docker:
+      - image: cimg/python:3.8.0
+
+aliases:
+  update-helm-params: &update-helm-params
+    parameters:
+      argo_application:
+        type: string
+        default: $CIRCLE_PROJECT_REPONAME
+      values_file:
+        type: string
+        default: ""
+      manifest_branch:
+        type: string
+        default: 'master'
+      image_tag_yaml_path:
+        type: string
+        default: ''
+      image_tag:
+        type: string
+      commit_tag_name:
+        type: string
+        default: ""
+      environment:
+        type: string
+      region:
+        type: string
+      ssh_key_fingerprint:
+        type: string
+      git_email:
+        type: string
+        default: ""
+      git_name:
+        type: string
+        default: "CircleCI Gitops Orb"
+      argocd_url:
+        type: string
+        default: ""
+      argocd_token: 
+        type: string
+        default: ""
+      yq_version: 
+        type: string
+        default: "v4.6.2"
+      wait_for: 
+        type: integer
+        default: 600
+
+commands:
+  install-python-dependencies:
+    description: "Install python dependencies"
+    steps:
+      - run:
+          name: "Install python dependencies"
+          command: |
+            pip3 install --upgrade pip
+            pip3 install requests
+
+  update_helm:
+    <<: *update-helm-params
```

Comment:
noob question, do you need to close the opening `<<` here?

Reply:
yes, its for injecting `update-helm-params`